.github/workflows/workflow.yml

```
@@ -20,9 +20,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
     - name: Setup Node.js 20.x
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
```

```suggestion
      uses: actions/setup-node@v3
```

Somehow `actions/setup-node` is also bumped in the last commit https://github.com/actions/cache/pull/1231/commits/dbe53203465e844da30e801ee1d2cd43671f63f4. But this change didn't belong to the current default branch (in upstream repo), thus I propose to revert it.
https://github.com/actions/cache/blob/a2ed59d39b352305bdd2f628719a53b2cc4f9613/.github/workflows/workflow.yml#L24-L25

---

Thanks, which resolving the conflict I seem to have accidentally bumped it as the latest Node 20-based version is v4 series (reverted it, but I don't currently see other PRs bumping it).

---

I wonder why this repo didn't use a `dependabot.yml` to keep GitHub Actions up-to-date.